project 3/CODE/index.html

```diff
@@ -0,0 +1,140 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css?family=Quicksand|Ubuntu:700" rel="stylesheet">
+    <title>Timeline</title>
+</head>
+
+<body>
+
+    <div id="hour6am">
+        <a href="#" class="button-close">&times;</a>
```

Comment:
This element needed to be styled to be visible, especially on the top and bottom of the page.